package/cloudshell/iac/terraform/__init__.py

```diff
@@ -2,5 +2,5 @@
 from pkgutil import extend_path
 __path__ = extend_path(__path__, __name__)
 
-from .models.config import TerraformShellConfig
-from .terraform_shell import TerraformShell
+# from .models.config import TerraformShellConfig
+# from .terraform_shell import TerraformShell
```

Comment:
Why this is commented out?
This 2 lines are important and commenting them is a breaking change...